tests/handlers/aws/test_handler.py

```diff
@@ -399,6 +399,8 @@ def test_lambda_handler_noop(self) -> None:
         with self.subTest("no originalEventSourceARN in messageAttributes"):
             ctx = ContextMock()
             os.environ["S3_CONFIG_FILE"] = "s3://s3_config_file_bucket/s3_config_file_object_key"
+            os.environ["SQS_REPLAY_URL"] = "https://sqs.us-east-2.amazonaws.com/123456789012/replay_queue"
+            os.environ["SQS_CONTINUE_URL"] = "https://sqs.us-east-2.amazonaws.com/123456789012/continue_queue"
             lambda_event = deepcopy(_dummy_lambda_event)
             del lambda_event["Records"][0]["messageAttributes"]["originalEventSourceARN"]
             assert handler(lambda_event, ctx) == "completed"  # type:ignore
```

Comment:
Struggling to understand why this assertion is completed?

The originalEventSourceARN == arn:aws:sqs:eu-central-1:123456789:sqs-queue" but here we have a different region. Should we raise another status?

Reply:
We are not using the `originalEventSourceARN` attribute here, but the `arn:aws:sqs:eu-central-1:123456789:s3-sqs-queue` instead. And the queues are not being used... But that is a great point. I suspect with would fail with different regions, but it is never being tested in this file (there are no timeout tests or misconfigured ids here, so no reason to use those queues). I had to add the line because of this part:
https://github.com/elastic/elastic-serverless-forwarder/blob/9c6d224137f94279970e8f3a9949c5b67ad32930/handlers/aws/handler.py#L135-L136
I will fix this as it does not make sense indeed